common/technology/technologies/20_military.txt

```diff
@@ -61,6 +61,7 @@ mandatory_service = {
 	
 	modifier = {
 		state_conscription_rate_mult = 0.2
+		state_building_conscription_center_max_level_add = 5
```

Comment:
Huge balance implications for professional army and peasant levy law. It is a really big buff to the amount of conscription it can do. 400% for peasant levy and 100% for professional army at the end of the techs.

In addition this doesn't buff the conscription heavy laws not quite as much because it requires really large population centers to make use of it.

Reply:
All of these would require large population centers to make use of them no? This is intended to work with the topline of recruitable troops from a major population center in the mid-late game, and take into consideration the fact that there are many more +conscription rate laws from home defence to push these limits.

Professional army also almost never reaches their maximum conscription center potential without national guard, but then can paradoxically end up pushing it way overboard. This just helps remove some upper limiters. Also, UHW changes those laws so the numbers are a bit different and overall conscription was heavily nerfed in their mod without Mass Conscription or National Militia. This helps support conscription rates from those other laws even more as a result.

Reply:
oki